eth/handler.go

```diff
@@ -175,7 +175,9 @@ func newHandler(config *handlerConfig) (*handler, error) {
 		fullBlock, snapBlock := h.chain.CurrentBlock(), h.chain.CurrentSnapBlock()
 
 		if fullBlock.Number.Uint64() == 0 && snapBlock.Number.Uint64() > 0 {
```

Comment:
Wouldn't it be better to comment out this entire if block?
Those uncommented warning logs could mislead us when we are debugging.

Reply:
Makes sense, updated.